dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentApp/src/concurrent/mp/fat/web/MPConcurrentTestServlet.java

```diff
@@ -5464,6 +5469,7 @@ public void testWithContextCapture_CompletableFuture_serverConfig() throws Excep
      * than a managed executor.
      */
     @Test
+    @SkipForRepeat(MPContextProp11RepeatAction.ID)
```

Comment:
@njr-11 This test fails with version 1.1, when you review please decide if a follow up change is needed to fix that.

Reply:
Again, the difference is expected. See previous comments.